SPEC.md

```diff
@@ -2114,6 +2115,42 @@ task example {
 Supported units are KiloByte ("K", "KB"), MegaByte ("M", "MB"), GigaByte ("G", "GB"), TeraByte ("T", "TB") as well as their [binary version](https://en.wikipedia.org/wiki/Binary_prefix) "Ki" ("KiB"), "Mi" ("MiB"), "Gi" ("GiB"), "Ti" ("TiB").
 Default unit is Bytes ("B").
 
+
+## String sub(String, String, String)
+
+Given 3 String parameters `input`, `pattern`, `replace`, this function will replace any occurence matching `pattern` in `input` by `replace`.
+`pattern` is expected to be a [regular expression](https://en.wikipedia.org/wiki/Regular_expression). There are different implemetations of regular expression evaluators, different sets of rules etc... This choice is left to the implementor.
```

Comment:
If not, how much more specific do we want to be ? I personally am no regex expert and I don't have any really strong opinion about which standard we should encourage/enforce :/